dns_discover.go

```diff
@@ -35,7 +35,7 @@ func discoverDNS(domain string, port int) (servers []string, ttl time.Duration,
 		}
 		for _, instance := range instances {
 			// format the service URL
-			servers = append(servers, fmt.Sprintf("http://%s:%d/eureka/v2", instance, port))
+			servers = append(servers, fmt.Sprintf("http://%s:%d/%s", instance, port, context))
```

Comment:
- [ ] Per my comment above, if we can tolerate an empty URL base, we should not include the trailing slash.

Reply:
See reply above, empty URL base is not possible.